goaviatrix/security_domain.go

```diff
@@ -207,6 +222,31 @@ func (c *Client) DisableIntraDomainInspection(ctx context.Context, intraDomainIn
 	return c.PostAPIContext(ctx, params["action"], params, BasicCheck)
 }
 
+func (c *Client) GetAllSecurityDomains() ([]DomainDetails, error) {
```

Comment:
I know all the old functions were not renamed when we changed from Security Domain to Network Domain. Should new functions added use Network Domain or will they all get changed together in the future? @shuoliugit @zhiqiangjin

Reply:
They all get changed together in the future.